cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -115,7 +141,19 @@ def _provisioner(template, provisioning_data, delayed=None):
     return _provisioner
 
 
+def check_all_tabs(provision_request, provider):
+    view = navigate_to(provision_request, "Details")
+
+    for name in provider.provisioning_dialog_widget_names:
```

Comment:
I'm unsure about defining these in the way that they have been.

We can read widget_names property on a view, which will return all the widgets.

If we want to test a specific set of those widget names, I think that should be defined in the test as the list/set of names - not on the provider itself.

Reply:
@mshriver that's what @jaryn had originally, but I wasn't crazy about it because of the lengthy `if/elif/else` block that it required for dealing with the differing providers. I suggested he put it in a data yaml file or a constant dict, but @jaryn liked this approach more.